### ISSUE TITLE
insufficient color contrast for error notification

### ISSUE THREAD

**Issue by robfentress** [ucfcdl/UDOIT#107]
The color contrast for the panel title (.panel-title) on error notifications is insufficient. It is considered "normal" text at 16px with a `font-weight` of 500 with a foreground `color` of `#a94442` and a `background-color` of `#ebccd1`.  This results in a contrast ratio of 3.92:1, which is below 4.5:1 ratio specified by WCAG 2.0 AA.

![screen shot 2016-02-24 at 4 32 10 pm](https://cloud.githubusercontent.com/assets/9356499/13301826/e0961448-db14-11e5-9807-79628a676b90.png)

**Comment by bagofarms** [comment]
The irony is strong with this one :-).  Adding the "High priority" text to the title, and we will get to this very soon.

**Comment by techrangerfj** [comment]
This was addressed in pull request #87
